pkg/apis/core/validation/validation_test.go

```diff
@@ -780,6 +889,22 @@ func TestValidationOptionsForPersistentVolume(t *testing.T) {
 	}
 }
 
+func getCSIVolumeWithSecret(pv *core.PersistentVolume, secret *core.SecretReference, secretfield string) *core.PersistentVolume {
+	pvCopy := pv.DeepCopy()
+	if secret != nil {
+		switch secretfield {
```

Comment:
add a default case that panics so a typo doesn't turn into a no-op

Reply:
sure, done